text/2020-09-28-memory-trace.md

```diff
@@ -0,0 +1,131 @@
+# Fully instrumented memory trace
+
+## Summary
+
+Make TiKV fully instrumented to provide accurate and details memory trace.
+
+## Motivation
+
+Suspicious memory increase has been a big challenge for TiKV. We don't have a
```

Comment:
Is this heap memory or does the stack run out as well?

Reply:
Heap memory.